charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.36.0
+version: 9.36.1
```

Comment:
IMO we don't need to rev the chart version, as we aren't adding or removing or changing any functionality. We can drop this change.

Reply:
as per the [comment](https://github.com/kubernetes/autoscaler/pull/7317#discussion_r1783194164), we need to change the chart version.